src/Compilers/Core/Portable/Diagnostic/CommonDiagnosticComparer.cs

```diff
@@ -14,26 +14,14 @@ internal sealed class CommonDiagnosticComparer : IEqualityComparer<Diagnostic>
     {
         public bool Equals(Diagnostic x, Diagnostic y)
         {
-            if (object.ReferenceEquals(x, y))
-            {
-                return true;
-            }
-
-            if (x == null || y == null)
-            {
-                return false;
-            }
-
+            if (object.ReferenceEquals(x, y))  return true;
+            if (x == null || y == null)        return false;
```

Comment:
There are a couple of issues here:

The first is that single line `if` statements aren't allowed under the [.NET foundation coding style](https://github.com/dotnet/corefx/wiki/Contributing#c-coding-style).  This is the style that Roslyn is following now for all of its code.  

The second is that in general we are going to be reluctant to take style only changes unless it either:

1. Significantly improves the readability of the code in question. 
2. Changes code to conform to the stated guidelines which previously did not.